salon_creator/lib/widgets/custom_textfield.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:flutter/material.dart';
+import 'package:salon_creator/widgets/custom_label.dart';
+
+Widget customTextFieldWithLabel({
```

Comment:
Looks like this is implemented as a simple function instead of a widget. Any reasons for doing this?